src/components/views/elements/MemberEventListSummary.js

```diff
@@ -144,87 +144,105 @@ module.exports = React.createClass({
         );
     },
 
-    render: function() {
-        let summary = null;
+    shouldComponentUpdate: function(nextProps, nextState) {
+        return (
+            nextProps.events.length !== this.props.events.length ||
```

Comment:
> this is going to mean that, when the summary is expanded, any updates to the eventTiles in children won't be rerendered. I don't know if that's a problem in practice, but it certainly sounds like a potential future problem.

Indeed, but I don't believe this is a problem in practice. Member events don't change after they have been rendered. I agree that this could be very tricky to debug in the future, so I would be up for always returning `true` if the summary is expanded, leaving it up to the children to decide if they want to re-render or not.